x-pack/plugins/security_solution/public/overview/components/entity_analytics/host_risk_score/index.tsx

```diff
@@ -38,19 +28,17 @@ import { useGlobalTime } from '../../../../common/containers/use_global_time';
 import { InspectButtonContainer } from '../../../../common/components/inspect';
 import { useQueryToggle } from '../../../../common/containers/query_toggle';
 import { hostsActions } from '../../../../hosts/store';
-import { useCheckSignalIndex } from '../../../../detections/containers/detection_engine/alerts/use_check_signal_index';
 import { RiskScoreDonutChart } from '../common/risk_score_donut_chart';
 import { BasicTableWithoutBorderBottom } from '../common/basic_table_without_border_bottom';
-import { useEnableHostRiskFromUrl } from '../../../../common/hooks/use_enable_host_risk_from_url';
+import { RISKY_HOSTS_EXTERNAL_DOC_LINK } from '../../../../../common/constants';
+import { EntityAnalyticsHostRiskScoreDisable } from '../../../../common/components/risk_score/risk_score_disabled/host_risk_score_disabled';
+import { RiskScoreHeaderTitle } from '../../../../common/components/risk_score/risk_score_onboarding/risk_score_header_title';
+import { RiskScoresNoDataDetected } from '../../../../common/components/risk_score/risk_score_onboarding/risk_score_no_data_detected';
 
 const TABLE_QUERY_ID = 'hostRiskDashboardTable';
 
-const IconWrapper = styled.span`
-  margin-left: ${({ theme }) => theme.eui.euiSizeS};
-`;
-
-export const EntityAnalyticsHostRiskScores = () => {
-  const { deleteQuery, setQuery, to, from } = useGlobalTime();
+const EntityAnalyticsHostRiskScoresComponent = () => {
```

Comment:
most of these UI components can be combined too with the `entityType` prop, let's do that in a follow up